### ISSUE TITLE
External links

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/page-constructor#485]
Need to state somehow whether a link leads to external resource or not.

The components where this feature definitely should be:

1. `Content` sub-block
1. `ExtendedFeatures` block
2. `ShareListItem` (components)

Probably, there are other link elements that should be marked as external

**Comment by Kyzyl-ool** [comment]
Decided to use as visual warning the arrow icon (see [here](https://preview.yandexcloud.dev/page-constructor/?path=/story/navigation-navigation--default-navigation)) and as warning for the screenreaders – `aria-describedby` attribute.
